src/commands/synthetics/command.ts

```diff
@@ -275,14 +285,15 @@ export class RunTestCommand extends Command {
 RunTestCommand.addPath('synthetics', 'run-tests')
 RunTestCommand.addOption('apiKey', Command.String('--apiKey'))
 RunTestCommand.addOption('appKey', Command.String('--appKey'))
-RunTestCommand.addOption('failOnCriticalErrors', Command.Boolean('--failOnCriticalErrors'))
 RunTestCommand.addOption('configPath', Command.String('--config'))
 RunTestCommand.addOption('datadogSite', Command.String('--datadogSite'))
-RunTestCommand.addOption('files', Command.Array('-f,--files'))
+RunTestCommand.addOption('failOnCriticalErrors', Command.Boolean('--failOnCriticalErrors'))
 RunTestCommand.addOption('failOnTimeout', Command.Boolean('--failOnTimeout'))
+RunTestCommand.addOption('files', Command.Array('-f,--files'))
+RunTestCommand.addOption('jUnitReport', Command.String('-j,--jUnitReport'))
 RunTestCommand.addOption('publicIds', Command.Array('-p,--public-id'))
-RunTestCommand.addOption('testSearchQuery', Command.String('-s,--search'))
+RunTestCommand.addOption('runName', Command.String('-n,--runName'))
 RunTestCommand.addOption('subdomain', Command.Boolean('--subdomain'))
+RunTestCommand.addOption('testSearchQuery', Command.String('-s,--search'))
 RunTestCommand.addOption('tunnel', Command.Boolean('-t,--tunnel'))
-RunTestCommand.addOption('jUnitReport', Command.String('-j,--jUnitReport'))
-RunTestCommand.addOption('runName', Command.String('-n,--runName'))
+RunTestCommand.addOption('variableStrings', Command.Array('-v,--variables'))
```

Comment:
nit: singular `variable` as I think they are only passed one-by-one (`... --variable A=1 --variable B=2`)
```suggestion
RunTestCommand.addOption('variableStrings', Command.Array('-v,--variable'))
```